guides/source/working_with_javascript_in_rails.md

```diff
@@ -24,11 +24,11 @@ In order to understand Ajax, you must first understand what a web browser does
 normally.
 
 When you type `http://localhost:3000` into your browser's address bar and hit
-'Go,' the browser (your 'client') makes a request to the server. It parses the
+'Go', the browser (your 'client') makes a request to the server. It parses the
 response, then fetches all associated assets, like JavaScript files,
 stylesheets and images. It then assembles the page. If you click a link, it
 does the same process: fetch the page, fetch the assets, put it all together,
-show you the results. This is called the 'request response cycle.'
+show you the results. This is called the 'Request-Response Cycle'.
```

Comment:
Any reason for the change? Request/Response are not constants here. Ex: See other uses in the guide. 

Rest looks good.

Reply:
Thanks for the reviewing. I'll restore it, soon.